src/images/extract.py

```diff
@@ -38,7 +38,9 @@
 
 IMG_URL_PREFIX = config.IMAGES_URL_PREFIX
 
-WIKIPEDIA_URL = "https://{}.wikipedia.org".format(config.LANGUAGE)
+WIKIPEDIA_URL = "https://%s.wikipedia.org" % config.LANGUAGE
+
+MEDIAWIKI_URL_PREFIX_LANG = "//upload.wikimedia.org/wikipedia/%s/" % config.LANGUAGE
```

Comment:
We prefer using `.format()`

Reply:
Ahh ok, I changed it to have a standard in the whole file because it was the only place where a string was formatted using `.format()`, but there is no problem and I'll proceed to change it :)

Reply:
Great, thanks!